app/uk/gov/hmrc/eeittadminfrontend/config/ApplicationLoader.scala

```diff
@@ -247,13 +249,12 @@ trait ApplicationModule extends BuiltInComponents
 
   val eeittUrl: String = s"${baseUrl("eeittadmin")}/eeitt"
 
-
-
+  val clientID = configuration.getString("ClientID")
```

Comment:
There is bunch of nice libraries which allows to create configs in type safe manner. 
We could consider to adopt for example this one: https://github.com/pureconfig/pureconfig